src/special/package.js

```diff
@@ -0,0 +1,30 @@
+export default function parsePackageJson(content, filepath, deps) {
+  // Mark dependencies as "used" if they are used in a script in package.json.
+  if (filepath.match(new RegExp('(\\/|\\\\)package.json$'))) {  // Escaping is fun!
+    // Grab just the scripts section.
+    const scripts = JSON.parse(content).scripts;
+
+    if (scripts) {
+      const scriptKeys = Object.keys(scripts);
```

Comment:
You could use [`lodash.values()`](https://lodash.com/docs/#values) to get values directly.